requirements.txt

```diff
@@ -3,3 +3,4 @@ git+https://github.com/rkdarst/sphinx-copybutton.git@exclude-unselectable-3
 sphinx-lesson
 sphinx_rtd_theme_ext_color_contrast
 https://github.com/coderefinery/sphinx-coderefinery-branding/archive/master.zip
+https://github.com/rkdarst/sphinx_pyppeteer_builder/archive/refs/heads/main.zip
```

Comment:
Should this be installed as part of the workflow yaml file, to not pull it in for every requirements install?
(since I'm not sure about the stability of it long term, I'll probably delete the repo eventually...)

I'm not sure, though...

Reply:
I agree - I will fix.